### PR TITLE
Fix Docker Build (#9)

### PR DESCRIPTION
**Pull request by maxirus**
## Fixed
- #9 
- Axios Critical Sev Vuln

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,17 @@
+FROM node:alpine as build
+WORKDIR /build
+COPY ./ ./
+RUN npm install gulp
+RUN npm run dist
+
 FROM node:alpine
+# TODO: This needed?
```
Comment (tborychowski) [inline Dockerfile:8]: it was needed when I wrote that. Can you check or remove the comment?

File: Dockerfile [hunk 2]
```diff
@@ -1,9 +1,17 @@
+FROM node:alpine as build
+WORKDIR /build
+COPY ./ ./
+RUN npm install gulp
+RUN npm run dist
```
Comment (tborychowski) [inline Dockerfile:5]: what's the benefit of running a build before creating the image (as opposed to "during")?
I've had some issues with this approach on another project.

Comment (tborychowski) [reply Dockerfile:5]: no matter, if it works, it works :-)

Comment (maxirus) [reply Dockerfile:5]: It wasn't actually building before as the public/ folder wasn't available. If you instead ran `npm run dist` in a single stage, you'd bring unwanted files/folders in tow with the release image. By splitting it into multi-stage, you're able to copy only the items you need from the `build` stage.

File: Dockerfile (hunk 1 above)
Comment (maxirus) [reply Dockerfile:8]: I couldn't find `curl` used anywhere in your code when I search for it. I suspect it was simply a remnant from debugging API calls.

**Pull request merged**